Formula/radare2.rb

```diff
@@ -90,6 +107,11 @@ def install
       inreplace "libr/lang/p/Makefile", "R2_PLUGIN_PATH=", "#R2_PLUGIN_PATH="
       inreplace "Makefile", "LUAPKG=", "#LUAPKG="
       inreplace "Makefile", "${DESTDIR}$$_LUADIR", "#{lib}/lua/#{lua_version}"
+      # fix some awkwardness
+      inreplace "Makefile" do |s|
+        s.gsub! "cp -rf python/r_*.py python/*.$$E ${PYTHON_INSTALL_DIR}", "cp -rf python/r_*.py python/*.$$E ${PYTHON_INSTALL_DIR} || true"
```

Comment:
Forcing `|| true` isn't really an acceptable fix for a bug like this. Has it been reported upstream? Do we know if it's been fixed in HEAD?